yarn/src/main/scala/org/apache/spark/deploy/yarn/YarnAllocator.scala

```diff
@@ -353,7 +353,7 @@ private[yarn] class YarnAllocator(
 
     } else if (missing < 0) {
       val numToCancel = math.min(numPendingAllocate, -missing)
-      logInfo(s"Canceling requests for $numToCancel executor containers")
+      logInfo(s"Canceled requests for $numToCancel executor container(s)")
```

Comment:
Ok, this can be only me... :)
"Driver requested a total number of 614 executor(s)". This means to me that the Driver already requested 614 executors. When I see the "Canceling requests for 5 executor containers" message, my interpretation is that it is cancelling 5 executors from the 614 already requested ones which is not true.